docker/Dockerfile.ci_adreno

```diff
@@ -22,10 +22,10 @@ COPY utils/apt-install-and-clear.sh /usr/local/bin/apt-install-and-clear
 
 # Android SDK
 COPY install/ubuntu_install_androidsdk.sh /install/ubuntu_install_androidsdk.sh
-RUN bash /install/ubuntu_install_androidsdk.sh
+RUN bash /install/ubuntu_install_androidsdk.sh 25.2.9519653 25
 ENV ANDROID_HOME=/opt/android-sdk-linux
-ENV ANDROID_NDK_HOME=/opt/android-sdk-linux/ndk/21.3.6528147
-ENV ANDROID_NDK_MAJOR=21
+ENV ANDROID_NDK_HOME=/opt/android-sdk-linux/ndk/25.2.9519653
+ENV ANDROID_NDK_MAJOR=25
```

Comment:
Is it necessary to specify these variables here? I see that they will be added to `/etc/profile` in `ubuntu_install_androidsdk.sh`.